src/view/src/ContentView.c

```diff
@@ -40,6 +40,7 @@ STATUS createContentView(UINT32 maxItemCount, UINT64 bufferDuration, ContentView
     pContentView->removeCallbackFunc = removeCallbackFunc;
     pContentView->itemBufferCount = maxItemCount;
     pContentView->bufferDuration = bufferDuration;
+    pContentView->bufferOverflowStrategy = DROP_UNTIL_FRAGMENT_START;

```

Comment:
I am kind of split on this but we should perhaps take this flag in and make decisions about this in the client.

Reply:
Also, I don't think we need to provide backward compatibility for this as we have no clients using content view directly.